src/Generator/CodeGenerator.php

```diff
@@ -192,14 +193,24 @@ public function writeEnumeratorAccessorsForClass(ReflectionClass $class): array
     }
 
     /**
-     * @param Enumerator $enumerator
-     * @return string
      * @throws \ReflectionException
      * @throws \Throwable
+     * @throws ReferencedClassNotFoundException
+     * @param  Enumerator          $enumerator
+     * @param  PropertyInformation $info
+     * @return string
      */
-    public function generateEnumeratorAccessors(Enumerator $enumerator)
+    public function generateEnumeratorAccessors(Enumerator $enumerator, PropertyInformation $info)
```

Comment:
This is a BC break as the method is public and the class is not marked as internal

Reply:
The only reason why these methods are public is to have them unit testable.

Reply:
That doesn't change the fact that they are publicly available. Can you add `@internal` to the method to prevent this from happening in the future?

Just because _we_ consider it something only used in tests, doesn't mean people using our package consider this (how can they know?).

Reply:
I'm not disagreeing with you. It's just that in most previous commits, the same 'mistake' was made.

Since @hipio is working on some code style fixes in the generator, maybe he can add these in his PR?

Reply:
I can do that, for sure, what exactly would you want me to change in the next major @iltar?

Reply:
Just mark this method as `@internal` (all methods that are public for tests only)